### PR TITLE
[devtools-translate] Russian - What's New in DevTools (Chrome 102)

### PR DESCRIPTION
**Pull request by solarrust**
Fixes: #2805

### CODE REVIEW

**Comment by jecfish** [comment]
Over to @kateryna-prokopenko and @webmaxru for review!

File: site/ru/blog/new-in-devtools-102/index.md [hunk 1]
```diff
@@ -5,148 +5,182 @@ authors:
...
+После завершения записи вы получите обзор производительности на панели **Insights**. Кликните на любой из пунктов отчёта (например, Render blocking request, Layout shift), чтобы определить проблему и возможные способы её устранения.
 
 <!-- Go to the **Performance insights** panel [documentation](/docs/devtools/performance-insights/) to learn more with the step-by-step tutorial.  -->
+Читайте [документацию](/docs/devtools/performance-insights/) о панели **Performance insights** чтобы узнать больше в пошаговом руководстве.
 
 <!-- This is a preview feature to help web developers (especially non-performance experts) to identify and fix potential performance issues. Our team is actively working on this feature and we are looking for your [feedback](https://crbug.com/1270700) for further enhancements. -->
+Это ранний доступ к функции, призванной помочь веб-разработчикам (особенно не специалистам в вопросе производительности) выявить и устранить потенциальные проблемы с производительностью. Наша команда все еще активно работает над этой новой функцией. Мы ждем ваши [отзывы](https://crbug.com/1270700) для дальнейших улучшений.
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-102/index.md:47]:
```suggestion
Это ранний доступ к функции, призванной помочь веб-разработчикам (особенно не специалистам в вопросе производительности) выявить и устранить потенциальные проблемы с производительностью. Наша команда все еще активно работает над этой функцией. Мы ждем ваши [отзывы](https://crbug.com/1270700) для дальнейших улучшений.
```

File: site/ru/blog/new-in-devtools-102/index.md [hunk 2]
```diff
@@ -5,148 +5,182 @@ authors:
...
 
-Chromium issue: [1270700](https://crbug.com/1270700)
+Задача в трекере Chromium: [1270700](https://crbug.com/1270700)
 
 
 <!-- ## New shortcuts to emulate light and dark themes {: #emulation } -->
+## Новые горячие клавиши для эмуляции светлой и тёмной темы {: #emulation }
 
 <!-- You can now emulate the light and dark themes quicker (CSS media feature [prefers-color-scheme](https://web.dev/prefers-color-scheme/#the-prefers-color-scheme-media-query)) with the new shortcuts in the **Styles** pane. -->
+Теперь вы можете быстро переключать светлую и тёмную темы (медиафункция [prefers-color-scheme](https://web.dev/prefers-color-scheme/#the-prefers-color-scheme-media-query) в CSS) при помощи горячих клавиш в панели **Стили** (Styles).
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-102/index.md:56]:
```suggestion
Теперь вы можете быстро переключать светлую и тёмную темы (CSS медиафункция [prefers-color-scheme](https://web.dev/prefers-color-scheme/#the-prefers-color-scheme-media-query)) при помощи горячих клавиш в панели **Стили** (Styles).
```

File: site/ru/blog/new-in-devtools-102/index.md [hunk 3]
```diff
@@ -5,148 +5,182 @@ authors:
...
 
 <!-- Previously, it took more steps to [emulate themes](/docs/devtools/rendering/emulate-css/) in the **Rendering** tab.   -->
+До этого требовалось больше действий для включения [эмуляции темы](/docs/devtools/rendering/emulate-css/) во вкладке **Отрисовка** (Rendering).
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-102/index.md:59]:
```suggestion
До этого требовалось больше действий для [эмуляции темы](/docs/devtools/rendering/emulate-css/) во вкладке **Отрисовка** (Rendering).
```

File: site/ru/blog/new-in-devtools-102/index.md [hunk 4]
```diff
@@ -5,148 +5,182 @@ authors:
...
 
-Chromium issue: [1314299](https://crbug.com/1314299)
+Задача в трекере Chromium: [1314299](https://crbug.com/1314299)
 
 
 <!-- ## Improve security on the Network Preview tab {: #network-preview } -->
+## Улучшение безопасности на вкладке Предварительный просмотр (Preview) панели Сеть (Network) {: #network-preview }
 
 <!-- DevTools now apply the Content Security Policy (CSP) in the **Preview** tab in the **Network** panel. -->
+DevTools теперь применяет политику безопасности содержимого (CSP) на вкладке **Предварительный просмотр** (Preview) на панели **Сеть** (Network).
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-102/index.md:72]:
```suggestion
DevTools теперь применяет политику безопасности содержимого (CSP) на вкладке **Предварительный просмотр** (Preview) внутри вкладки **Сеть** (Network).
```

File: site/ru/blog/new-in-devtools-102/index.md [hunk 5]
```diff
@@ -5,148 +5,182 @@ authors:
...
 
 <!-- For example, the first screenshot shows a page that contains [mixed content](https://web.dev/what-is-mixed-content/). The page loads over a secure HTTPS connection, but the stylesheet loads over an insecure HTTP connection. -->
+Например, на первом скриншоте показана страница, содержащая [смешанный контент](https://web.dev/what-is-mixed-content/). Страница загружается через безопасное соединение HTTPS, но таблица стилей загружается через небезопасное соединение HTTP.
 
 <!-- The browser blocked the stylesheet request by default. However, when you opened the page via the **Preview** tab in the **Network** panel, the stylesheet was not blocked previously (hence the background turned into red). It is now blocked as you would expect (second screenshot). -->
+Браузер по умолчанию блокирует запрос таблицы стилей. Однако, когда вы открывали страницу через вкладку **Предварительный просмотр** (Preview) в панели **Сеть** (Network), таблица стилей ранее не блокировалась (поэтому фон становился красным). Теперь запрос блокируется, как и следовало ожидать (второй скриншот).
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-102/index.md:78]:
```suggestion
Браузер по умолчанию блокирует запрос таблицы стилей. Однако, когда вы открывали страницу через вкладку **Предварительный просмотр** (Preview) во вкладке **Сеть** (Network), таблица стилей ранее не блокировалась (поэтому фон становился красным). Теперь запрос блокируется, как и следовало ожидать (второй скриншот).
```

File: site/ru/blog/new-in-devtools-102/index.md [hunk 6]
```diff
@@ -5,148 +5,182 @@ authors:
...
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/jxqxoJYqWXGzj4V9aJaX.png", alt="Improve security on the Network Preview tab", width="800", height="488" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/jxqxoJYqWXGzj4V9aJaX.png", alt="Улучшение безопасности на вкладке Предварительный просмотр панели Сеть", width="800", height="488" %}
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-102/index.md:80]:
```suggestion
{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/jxqxoJYqWXGzj4V9aJaX.png", alt="Улучшение безопасности на вкладке Предварительный просмотр во вкладке Сеть", width="800", height="488" %}
```

File: site/ru/blog/new-in-devtools-102/index.md [hunk 7]
```diff
@@ -5,148 +5,182 @@ authors:
...
 
-Chromium issue: [833147](https://crbug.com/833147)
+Задача в трекере Chromium: [833147](https://crbug.com/833147)
 
 
 <!-- ## Improved reloading at breakpoint {: #debugger } -->
+## Улучшена перезагрузка в точке останова {: #debugger }
 
 <!-- The debugger now terminates script execution when reloading at breakpoint. -->
+Отладчик теперь завершает выполнение сценария при перезагрузке по точке останова.
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-102/index.md:91]:
```suggestion
Отладчик теперь завершает выполнение сценария при перезагрузке на точке останова.
```

File: site/ru/blog/new-in-devtools-102/index.md [hunk 8]
```diff
@@ -5,148 +5,182 @@ authors:
...
 
 <!-- For example, the script got into an endless loop previously when setting and reloading at the `ReactDOM` breakpoint in this [React demo](https://react-stuck.glitch.me/). The **Sources** panel broke due to the endless loop.  -->
+Например, ранее скрипт попадал в бесконечный цикл при установке и перезагрузке в точке останова `ReactDOM` в этом [демо-приложении](https://react-stuck.glitch.me/). Панель **Источники** (Sources) ломалась из-за бесконечного цикла.
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-102/index.md:94]:
```suggestion
Например, ранее скрипт попадал в бесконечный цикл при установке и перезагрузке в точке останова `ReactDOM` в этом [демо-приложении](https://react-stuck.glitch.me/). Вкладка **Источники** (Sources) ломалась из-за бесконечного цикла.
```

File: site/ru/blog/new-in-devtools-102/index.md [hunk 9]
```diff
@@ -5,148 +5,182 @@ authors:
...
 
 <!-- Continuing to execute JavaScript is causing a lot of trouble for developers and might leave the renderer in a broken state. This change aligns the debugging behavior with other browsers like Firefox. -->
+Продолжение выполнения JavaScript создает много проблем для разработчиков и может сломать рендер. Теперь отладчик отрабатывает в этой ситуации как в других браузерах, например, Firefox.
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-102/index.md:97]:
```suggestion
Продолжение выполнения JavaScript создает много проблем для разработчиков и может сломать рендерер. Это изменение обеспечивает соответствие поведения отладки с другими браузерами, такими как Firefox.
```

File: site/ru/blog/new-in-devtools-102/index.md [hunk 10]
```diff
@@ -5,148 +5,182 @@ authors:
...
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/gBtY4zD39SPizfcCGJJW.png", alt="Обработка ошибок выполнения сценария в консоли", width="800", height="487" %}
 
 {# https://chromium.googlesource.com/v8/v8/+/56cfdd68c731c53d016326b890b56b5c30098998 #}
 
-Chromium issue: [1295750](https://crbug.com/1295750)
+Задача в трекере Chromium: [1295750](https://crbug.com/1295750)
 
 
 <!-- ### Commit live expression with Enter {: #live-expression } -->
+### Выполните живое выражение с помощью Enter {: #live-expression }
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-102/index.md:123]:
```suggestion
### Выполнение живых выражений с помощью Enter {: #live-expression }
```

File: site/ru/blog/new-in-devtools-102/index.md [hunk 11]
```diff
@@ -5,148 +5,182 @@ authors:
...
 
 <!-- Once you finish typing a [live expression](/blog/new-in-devtools-70/#watch), you can click `Enter` to commit it. Previously, hitting Enter resulted in adding new lines. This is inconsistent with other parts of the DevTools.  -->
+После окончания набора [живого выражения](/blog/new-in-devtools-70/#watch) можете нажать `Enter` чтобы выполнить его. Раньше нажатие на Enter переводило курсор на новую строку. Это противоречило поведению в других инструментах DevTools.
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-102/index.md:126]:
```suggestion
После того, как вы закончите набирать [живое выражение](/blog/new-in-devtools-70/#watch), вы можете нажать `Enter`, чтобы выполнить его. Раньше нажатие на Enter приводило к добавлению новых строк. Это противоречило поведению в других инструментах DevTools.
```

File: site/ru/blog/new-in-devtools-102/index.md [hunk 12]
```diff
@@ -5,148 +5,182 @@ authors:
...
 
 <!-- To add a new line in the **live expression** editor, use `Shift` + `Enter` instead. -->
+Чтобы перейти на новую строку в редакторе **живых выражений** используйте `Shift` + `Enter`.
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-102/index.md:129]:
```suggestion
Чтобы перейти на новую строку в редакторе **живых выражений** используйте вместо этого `Shift` + `Enter`.
```

File: site/ru/blog/new-in-devtools-102/index.md [hunk 13]
```diff
@@ -5,148 +5,182 @@ authors:
...
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/yB7m2052mYzgsRgjIMvs.png", alt="Commit live expression with Enter", width="800", height="541" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/yB7m2052mYzgsRgjIMvs.png", alt="Выполните живое выражение с помощью Enter", width="800", height="541" %}
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-102/index.md:131]:
```suggestion
{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/yB7m2052mYzgsRgjIMvs.png", alt="Выполнение живых выражений с помощью Enter", width="800", height="541" %}
```

File: site/ru/blog/new-in-devtools-102/index.md [hunk 14]
```diff
@@ -5,148 +5,182 @@ authors:
...
 
-Chromium issue: [1257499](https://crbug.com/1257499)
+Задача в трекере Chromium: [1257499](https://crbug.com/1257499)
 
 
 <!-- ## Display inherited highlight pseudo-elements in the Styles pane {: #pseudo } -->
+## Отображение наследуемых псевдоэлементов выделения в панели Стили (Styles) {: #pseudo }
 
 <!-- View the inherited highlight pseudo-elements  (e.g. `::selection`, `::spelling-error`, `::grammar-error`, and `::highlight`) in the **Styles** pane. Previously, these rules were not displayed. -->
+Просматривайте наследуемые псевдоэлементы выделения (`::selection`, `::spelling-error`, `::grammar-error` и `::highlight`) в панели **Стили** (Styles). Раньше эти правила были не видны.
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-102/index.md:153]:
```suggestion
Просматривайте наследуемые псевдоэлементы выделения (`::selection`, `::spelling-error`, `::grammar-error` и `::highlight`) в панели **Стили** (Styles). Раньше эти правила не отображались. 
```

File: site/ru/blog/new-in-devtools-102/index.md [hunk 15]
```diff
@@ -5,148 +5,182 @@ authors:
...
 
 <!-- View the inherited highlight pseudo-elements  (e.g. `::selection`, `::spelling-error`, `::grammar-error`, and `::highlight`) in the **Styles** pane. Previously, these rules were not displayed. -->
+Просматривайте наследуемые псевдоэлементы выделения (`::selection`, `::spelling-error`, `::grammar-error` и `::highlight`) в панели **Стили** (Styles). Раньше эти правила были не видны. 
 
 <!-- As mentioned in the [specification](https://drafts.csswg.org/css-pseudo-4/#highlight-cascade), when multiple styles conflict, cascade determines the winning style. This new feature helps you understand the inheritance and priority of the rules. -->
+Как указано в [спецификации](https://drafts.csswg.org/css-pseudo-4/#highlight-cascade), когда несколько стилей конфликтуют, каскад определяет победивший стиль. Эта новая функция помогает понять наследование и приоритет правил.
 
 {% Aside %}
 <!-- At the moment, you need to run Chrome with the `--enable-blink-features=HighlightInheritance` flag to enable this feature. -->
+В данный момент вам нужно запустить Chrome в флагом `--enable-blink-features=HighlightInheritance` чтобы активировать эту функцию
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-102/index.md:160]:
```suggestion
В данный момент вам нужно запустить Chrome с флагом `--enable-blink-features=HighlightInheritance`, чтобы активировать эту функцию
```

File: site/ru/blog/new-in-devtools-102/index.md [hunk 16]
```diff
@@ -5,148 +5,182 @@ authors:
...
+## Другие важные моменты {: #misc }
 
 <!-- These are some noteworthy fixes in this release: -->
+Вот несколько заслуживающих внимания исправлений в этом выпуске:
 
 <!-- - The **Properties** pane now displays accessor properties with value by default. It was hidden mistakenly previously. ([1309087](https://crbug.com/1309087))
 - The **Styles** pane now properly shows the overridden `@support` rules as strikethrough. Previously, the rules weren’t strikethrough. ([1298025](https://crbug.com/1298025))
 - Fixed the CSS formatting logic in the **Sources** panel that caused multiple blank lines when editing CSS. ([1309588](https://crbug.com/1309588))
 - Cap the **Expand recursively** option of an object in the **Console** to maximum 100 so it does not go on forever for circular objects. ([1272450](https://crbug.com/1272450)) -->
+- На панели **Свойства** (Properties) теперь по умолчанию отображаются свойства-аксессоры вместе со значениями. Ранее оно по ошибке были скрыты. ([1309087](https://crbug.com/1309087))
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-102/index.md:180]:
```suggestion
- На панели **Свойства** (Properties) теперь по умолчанию отображаются свойства-аксессоры вместе со значениями. Ранее это было скрыто по ошибке. ([1309087](https://crbug.com/1309087))
```

File: site/ru/blog/new-in-devtools-102/index.md [hunk 17]
```diff
@@ -5,148 +5,182 @@ authors:
...
+- В панели **Стили** (Styles) переопределенные правила `@support` теперь правильно отображаются как зачеркнутые. Ранее правила не зачёркивались. ([1298025](https://crbug.com/1298025))
+-  Исправлена логика форматирования CSS в панели **Источники** (Sources), которая приводила к появлению множества пустых строк при редактировании CSS. ([1309588](https://crbug.com/1309588))
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-102/index.md:182]:
```suggestion
-  Исправлена логика форматирования CSS во вкладке **Источники** (Sources), которая приводила к появлению множества пустых строк при редактировании CSS. ([1309588](https://crbug.com/1309588))
```

File: site/ru/blog/new-in-devtools-102/index.md [hunk 18]
```diff
@@ -156,41 +190,50 @@ Chromium issue: [1024156](https://crbug.com/1024156)
 
 
 <!-- ## [Experimental] Copy CSS changes {: #copy } -->
+## [Эксперимент] Копирование изменений в CSS {: #copy }
 
 {% Aside %}
 <!-- To enable the experiment, check **Sync CSS changes in the Styles pane** under **Settings** > **Experiments**. -->
+Чтобы включить этот эксперимент поставьте галочку напротив **Sync CSS changes in the Styles pane** по пути **Настройки** (Settings) > **Эксперименты** (Experiments).
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-102/index.md:197]:
```suggestion
Чтобы включить этот эксперимент поставьте галочку напротив **Sync CSS changes in the Styles pane** в **Настройки** (Settings) > **Эксперименты** (Experiments).
```

File: site/ru/blog/new-in-devtools-102/index.md [hunk 19]
```diff
@@ -156,41 +190,50 @@ Chromium issue: [1024156](https://crbug.com/1024156)
...
 {% endAside %}
 
 <!-- With this experiment, the **Styles** pane highlights your CSS changes in green. You can hover over the changed rules and click on the new copy button next to it to copy it. -->
+В этом эксперименте добавлена зелёная подсветка для внесённых вами изменений в CSS. Вы можете навести курсор на изменённое свойство и нажать на новую кнопку копирования чтобы скопировать его.
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-102/index.md:201]:
```suggestion
С этим экспериментом панель **Стили** (Styles) подсвечивает зелёным ваши CSS изменения. Вы можете навести курсор на изменённое свойство и нажать на новую кнопку копирования, чтобы скопировать его.
```

File: site/ru/blog/new-in-devtools-102/index.md [hunk 20]
```diff
@@ -156,41 +190,50 @@ Chromium issue: [1024156](https://crbug.com/1024156)
...
 
 <!-- Apart from that, you can copy all CSS changes across declarations by right-clicking on any rule, and selecting **Copy all CSS changes**. -->
+Кроме того, вы можете скопировать все изменения CSS, щелкнув правой кнопкой мыши на любом правиле и выбрав **Копировать все изменения кода CSS**.
 
 <!-- A new **Copy** button is added to the [Changes](/docs/devtools/changes/) tab as well to help you keep track and copy your CSS changes with ease! -->
+Новая кнопка **Copy** также добавлена на вкладку [Изменения](/docs/devtools/changes/) чтобы помочь вамлегко отслеживать и копировать изменения в CSS!
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-102/index.md:207]:
```suggestion
Новая кнопка **Copy** также добавлена на вкладку [Изменения](/docs/devtools/changes/), чтобы помочь вам легко отслеживать и копировать изменения в CSS!
```

File: site/ru/blog/new-in-devtools-102/index.md [hunk 21]
```diff
@@ -156,41 +190,50 @@ Chromium issue: [1024156](https://crbug.com/1024156)
...
-Chromium issue: [1268754](https://crbug.com/1268754)
+Задача в трекере Chromium: [1268754](https://crbug.com/1268754)
 
 
 <!-- ## [Experimental] Picking color outside of browser {: #color-picker } -->
+## [Эксперимент] Выбор цвета вне браузера {: #color-picker }
 
 {% Aside %}
 <!-- To enable the experiment, check **Enable color picking outside the browser window** under **Settings** > **Experiments**. -->
+Чтобы включить этот эксперимент поставьте галочку напротив **Enable color picking outside the browser window** по пути **Настройки** (Settings) > **Эксперименты** (Experiments).
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-102/index.md:223]:
```suggestion
Чтобы включить этот эксперимент поставьте галочку напротив **Enable color picking outside the browser window** в **Настройки** (Settings) > **Эксперименты** (Experiments).
```

File: site/ru/blog/new-in-devtools-102/index.md [hunk 22]
```diff
@@ -156,41 +190,50 @@ Chromium issue: [1024156](https://crbug.com/1024156)
...
 {% endAside %}
 
 <!-- Enable this experiment to pick a color outside of the browser with the color picker. Previously, you could only pick a color within the browser. -->
+Запустили эксперимент, позволяющий выбирать цвет за пределами окна браузера. Раньше вы могли захватывать цвет только в пределах окна браузера.
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-102/index.md:227]:
```suggestion
Запустите эксперимент, чтобы выбирать цвет за пределами окна браузера. Раньше вы могли захватывать цвет только в пределах окна браузера.
```

File: site/ru/blog/new-in-devtools-102/index.md [hunk 23]
```diff
@@ -156,41 +190,50 @@ Chromium issue: [1024156](https://crbug.com/1024156)
...
 
 <!-- In the **Styles** pane, click on any color preview to open the color picker. Use the eyedropper to pick color from anywhere.  -->
+В панели **Стили** (Styles) кликните на любое превью цвета чтобы открыть палитру цветов. Используйте пипетку, чтобы выбрать цвет из любого места.
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-102/index.md:230]:
```suggestion
В панели **Стили** (Styles) кликните на любое превью цвета, чтобы открыть палитру цветов. Используйте пипетку, чтобы выбрать цвет из любого места.
```

File: site/ru/blog/new-in-devtools-102/index.md (hunk 9 above)
Comment (solarrust) [reply site/ru/blog/new-in-devtools-102/index.md:97]: What is «рендерер»? 🤔 I think that in Russian it will be «рендеринг».

**Comment by jecfish** [comment]
Thanks Kateryna and Alena, over to @webmaxru for last review.

**Review by jecfish — APPROVED** [review]

**Pull request merged**